### ISSUE TITLE
Bullet Character

### ISSUE THREAD

**Issue by rctngle** [mwilliamson/mammoth.js#337]
Is it possible to preserve custom bullet characters in the converted html?

**Comment by hiepsieunhan** [comment]
Hey @mwilliamson, do you have any plan for implementing this one anytime soon?

**Comment by mwilliamson** [comment]
Nope, no plans! Having a minimal example document, the HTML that's currently generated, and the expected HTML would help, but in general I don't do much development on Mammoth anyway at the moment.

**Comment by rctngle** [comment]
@hiepsieunhan if its helpful, we worked around this by building the custom bullets into the character style in Word, then using style mapping to add a class to the list. Then we could re-add the custom bullets with css.
